bilal-waheed/blockchain/cli-tool/broadcast-tx

bilal-waheed opened the pull request:
## Overview

1. This PR contains the function for sponsored transactions. 
2. Readme file has been updated to incorporate the command syntax for sponsored transactions.
3. Minor changes in reading nonces in other transaction functions.

bilalanees98 reviewed (changes requested):
https://github.com/bilal-waheed/stacks/pull/1 will have some carried over changes. 
Will need to be reviewed once those are made.

bilal-waheed committed:
refactor: create constants.js file, create sendRawTransaction helper function, add npm script sto start the tool, convert urls to dynamic

what: code refactoring

why: to fix static url, to improve the code structure and make it easy-to-extend

impact: medium

bilal-waheed committed:
refactor: update the readme file and incorporate constants in sponsoredTransaction

what: update readme file and refactor to incorporate constants in sponsoredTransaction.js

why: to make the code consistent

impact: low

bilalanees98 commented on src/rbfTransaction.js:
```diff
@@ -0,0 +1,48 @@
+};
+
+export const rbfTransaction = async () => {
+  const transactionData = await getRawTx(TXID);
```
destruct the needed properties from the object

bilalanees98 reviewed (changes requested):
Please update according to the mentioned comments

bilal-waheed committed:
refactor: refactor rbfTransaction.js to destruct rawTx

what: rbfTransaction.js refactor

why: to destruct raw tx to make code more readable

impact: low

bilalanees98 reviewed (approved):
I think the .DS_Store file can be added .gitignore. But other than that all good.

bilal-waheed committed:
Merge branch 'master' into bilal/blockchain/cli-tool/broadcastTx
